src/resources/generated.rs

```diff
@@ -19,13 +19,15 @@ pub mod core {
     pub mod cash_balance;
     pub mod charge;
     pub mod customer;
+    pub mod custom_unit_amount;
```

Comment:
Saw a comment at the top of this file saying this should be generated - I couldn't find where in the codegen that should happen though. Please let me know if this needs to be fixed a different way :)